### PR TITLE
Updated arscan.c to support case-insensitive scan.

### PR DESCRIPTION
**Pull request by rsbeckerca**
This fixes issue #1

### CODE REVIEW

File: arscan.c [hunk 1]
```diff
@@ -834,7 +834,7 @@ ar_name_equal (const char *name, const char *mem, int truncated)
       abort ();
 #else
 #ifdef _GUARDIAN_TARGET
-      return !strcasecmp (name, mem);
+      return strcasecmp (name, mem) == 0;
```
Comment (Jojo-Schmitz) [inline arscan.c:837]: Actually I like `!strcasecmp()` better

Comment (rsbeckerca) [reply arscan.c:837]: Ok, I'll revert.

**Comment by rsbeckerca** [comment]
I like git. Revert is so easy.

**Comment by Jojo-Schmitz** [comment]
Would we want to merge a PR with 3 commits, or rather have it `git push --force`d with just one?

**Comment by rsbeckerca** [comment]
> 
> 
> Would we want to merge a PR with 3 commits (with 3rd reverting 2nd), or rather have it `git push --force`d with just one, the 1st?

I’m going to do a merge squash so it really doesn’t matter how many are in the PR.

**Review by Jojo-Schmitz — APPROVED** [review]

**Comment by Jojo-Schmitz** [comment]
OK

**Comment by rsbeckerca** [comment]
> 
> 
> > Would we want to merge a PR with 3 commits (with 3rd reverting 2nd), or rather have it `git push --force`d with just one, the 1st?
> 
> I’m going to do a merge squash so it really doesn’t matter how many are in the PR.

Alternatively, a `git rebase -i --autosquash `_commit-id_, then a `git push --force` works as well. OpenSSL prefers to do both, which is somewhat confusing to me.

**Pull request merged**